Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -503,11 +503,16 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
         // Dark blurred background.
         blurView.autoPinEdgesToSuperviewEdges()
 
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .leading)
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+        leaveCallViewButton.autoPinEdge(toSuperviewEdge: .leading)
+
+        if #available(iOS 11, *) {
+            leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+            contactNameLabel.autoPinEdge(toSuperviewMargin: .top)
+        } else {
+            leaveCallViewButton.autoPin(toTopLayoutGuideOf: self, withInset: 0)
```

Comment:
I actually don't know why I had to do this. =/

Reply:
Please maker sure to test this on iPhone X too if you haven't already.